### PR TITLE
revise beacon user story to indicate health or mission data

### LINKED ISSUE #216: revise user story 2 to indicate beacon contains health or mission data
beacon should alternate between satellite health telemetry data & mission data

### PR DESCRIPTION
**Pull request by capsulecorplab**
closes #216

### CODE REVIEW

File: architecture/2-UserStories/2-ViewBeaconData.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+id: 2
+name: View Satellite Beacon Data
+actor: Ground Station Operator
+behavior: view satellite beacon data
+rationale: verify that satellite is operating nominally
+derivedFrom: []
+example: View satellite beacon data (health or mission data) to verify TLE's correspond with expected orbit profile and/or to validate that a mission mode was successful
```
Comment (kchiu002) [inline architecture/2-UserStories/2-ViewBeaconData.yaml:7]: Add "that" before "TLE's".
Remove apostrophe in "TLE's".

**Review by kchiu002 — COMMENTED** [review]
See line comment

File: architecture/2-UserStories/2-ViewBeaconData.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+id: 2
+name: View Satellite Beacon Data
+actor: Ground Station Operator
+behavior: view satellite beacon data (alternating between health & mission) data, received via UHF
```
Comment (kchiu002) [inline architecture/2-UserStories/2-ViewBeaconData.yaml:4]: Reads "data (alternating between health & mission) data" here

**Review by kchiu002 — COMMENTED** [review]
See line comment.

File: architecture/2-UserStories/2-ViewBeaconData.yaml (hunk 2 above)
Comment (capsulecorplab) [reply architecture/2-UserStories/2-ViewBeaconData.yaml:4]: Yeah, the idea is that the beacon data will alternate between transmitting both health data and mission data, as to offload having to request mission data manually. Though, the verbiage could use some refining

Comment (capsulecorplab) [reply architecture/2-UserStories/2-ViewBeaconData.yaml:4]: Or were you suggesting the 2nd "data" is a bit redundant?

Comment (kchiu002) [reply architecture/2-UserStories/2-ViewBeaconData.yaml:4]: I read it as "satellite beacon data data" if you removed the part in parenthesis with the part in parenthesis describing "satellite beacon data".  It looks like you just repeated "data". Place the second "data" within the parenthesis

File: architecture/2-UserStories/4.1-RequestSatelliteHealthData.yaml [hunk 3]
```diff
@@ -5,4 +5,4 @@ behavior: request satellite health data packet
 rationale: verify/validate AODS sensors & GPS data are within nominal parameters
 derivedFrom:
 - "2-UserStories/4-RequestTelemetryData.yaml"
-example: Request satellite health data packet to verify or validate AODS sensors and GPS data correspond to expected attitude and/or orbit profile based on pre-computed attitude or orbit propagation model
+example: Request satellite health data packet to verify or validate TLE data corresponds to expected orbit profile based on pre-computed orbit propagation model
```
Comment (kchiu002) [inline architecture/2-UserStories/4.1-RequestSatelliteHealthData.yaml:8]: change "corresponds" to "corresponding"

**Review by kchiu002 — COMMENTED** [review]
See line comment.

**Review by kchiu002 — APPROVED** [review]
LGTM

**Pull request merged**